sdk/keyvault/keyvault-admin/samples-dev/accessControlHelloWorld.ts

```diff
@@ -40,12 +40,11 @@ export async function main(): Promise<void> {
       ]
     }
   ];
-  let roleDefinition = await client.upsertRoleDefinition(
-    globalScope,
+  let roleDefinition = await client.upsertRoleDefinition(globalScope, {
     roleDefinitionName,
```

Comment:
Nit: you might want to make this a friendly name, like "Backup Officer" or something to show it's *meant* to be a display name.